Heron/ImportVector.cs

```diff
@@ -530,6 +505,40 @@ protected override void SolveInstance(IGH_DataAccess DA)
             DA.SetDataTree(8, gtype);
         }
 
+        private List<Layer> GetLayers(DataSource dataSource)
+        {
+            List<OSGeo.OGR.Layer> layerset = new List<OSGeo.OGR.Layer>();
+            List<int> fc = new List<int>();
```

Comment:
layerSet? Or another thought, we can tell this is a list of layers just by looking at this code, could we just call it layers?

Reply:
Looks like fc is unused too

Reply:
I like `layerSet`.  Just `layer`was originally avoided in order to not be confused with Rhino's layers.